packages/app-desktop/gui/ConfigScreen/filterSettingsByQuery.ts

```
@@ -0,0 +1,27 @@
+		for (const md of section.metadatas) {
+			const label = (typeof md.label === 'function' ? md.label(appType) : md.label) || '';
+			const desc = (typeof md.description === 'function' ? md.description(appType) : md.description) || '';
+			if (label.toLowerCase().includes(query.toLowerCase()) || desc.toLowerCase().includes(query.toLowerCase())) {
+				matchedKeys.push(md.key);
+			}
+		}
+	}
+	return matchedKeys;
+};
+
+export default filterSettingsByQuery;
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)